cmds/rush/exit_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"testing"
+)
+
+type test struct {
+	in  string
+	out string
+	ret int
+}
+
+var tests = []test{
+	{"exit\n", "", 0},
+	{"exit 77\n", "", 77},
+	{"exit 1 2 3\n", "Too many arguments\n", 0},
+	{"exit abcd\n", "Non numeric argument\n", 0},
+}
```

Comment:
I am thinking that we can test all of rush this way. I have not yet tried, but it should work for something like:

         {"echo 1 && false && echo 2 \n exit", "1", 0},

I'll have to add another field for stdout mind you.